packages/clerk-js/src/ui/signUp/SignUpStart.tsx

```diff
@@ -92,6 +92,9 @@ function _SignUpStart(): JSX.Element {
   };
 
   React.useLayoutEffect(() => {
+    if (Object.values(fields).filter(f => f === 'on').length) {
```

Comment:
🙃 Super minor, but would it make sense to move this guard inside `handleTokenFlow` so all token-related logic is colocated?

Reply:
Debated this myself. I eventually thought that `handleTokenFlow` should act and not think. But since you're mentioning it, I can certainly update. :smile:

Reply:
Its a minor change anyway, go with the one that you like best 🙃